build/preset-es2015.js

```diff
@@ -0,0 +1,6 @@
+const esModules = process.env.BABEL_ENV === 'es';
```

Comment:
It looks like `commonjs` is the default, so how about 

```js
const esOptions = process.env.BABEL_ENV === 'es' ? { modules: false } : {}
```

That way we don't have to hardcode `commonjs` and we can just use Babel defaults.